backend/app/controllers/books_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BooksController < ApplicationController
+  def new
+    @book = Book.new
+  end
+
+  def create
+    @book = Book.new(book_params)
+
+    if @book.save
+      render json: @book, status: :created, location: @book
```

Comment:
It would be better to use a serializer / presenter service to control which parts of the Book model are sent over to the frontend. 

But since this is a simple app - this is more of a `nit` for now